### DISCUSSION TITLE
Block trackers

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1251] [Feature Requests]
I suggest to remove any trackers from forwarded emails, similar to what is implemented in the email protection service from DuckDuckGo.

I suggest to implement this feature as follows:
- Block specific domains by special lists ([example](https://github.com/OneClickLab/ugly-email-trackers))
- Block ALL external images with suspicious sizes like 1px, 2px, display:hidden, display:none, etc.
- Block all links like click.*, mailtrack.*, etc.
- Remove UTM-like parameters from all links (example of such parameters is given [here](https://github.com/AdguardTeam/AdguardFilters/blob/master/TrackParamFilter/sections/general_url.txt))

Also, a nice bonus would be to add to the email itself information on how many trackers are blocked.

This can significantly expand service's functionality and make it more private.

**Comment by CorsairVelo** [comment]
I support the idea. Since Simplelogin is now joined with Protonmail, perhaps you could leverage some of Proton mail's tracker removal tech?

**Reply by nguyenkims** [reply]
Using the Proton tracker detection in SimpleLogin is indeed in our roadmap and we hope to be able to implement it soon, please stay tuned!

**Reply by CorsairVelo** [reply]
Oh awesome!  Can't wait to hear more. I'm new to Simplelogin but am extremely happy with it.  (Been on Proton for 2 years).
